baseapp-chats/baseapp_chats/graphql/mutations.py

```diff
@@ -168,6 +168,16 @@ def mutate_and_get_payload(cls, root, info, profile_id, participants, is_group,
         room.participants_count = len(created_participants)
         room.save()
 
+        if is_group:
+            send_message(
+                message_type=Message.MessageType.SYSTEM_GENERATED,
+                room=room,
+                profile=None,
+                user=None,
+                content='{content_linked_profile_actor} created group "' + title + '"',
```

Comment:
maybe create {content_linked_profile_actor} and {content_linked_profile_receiver} constants? we may need to use it a lot

Reply:
@anicioalexandre so far the content will serve only for system messages which is now just being used for a new room with multiple people, right? So what's the purpose of having the `content_linked_profile_receiver`?  Maybe I'm missing something

Reply:
@vitorguima  in the near future, we will have other kinds of system messages, like `"{content_linked_profile_actor} added {content_linked_profile_receiver}"`